test/visualTest.ts

```diff
@@ -574,20 +574,72 @@ describe("EnhancedScatterChart", () => {
             });
 
             it("show", () => {
-                (<any>dataView.metadata.objects).outline.show = true;
                 visualBuilder.updateFlushAllD3Transitions(dataView);
 
                 visualBuilder.dots.forEach((element: HTMLElement) => {
-                    assertColorsMatch(element.style.fill, element.style.stroke, true);
+                    expect(element.style.strokeWidth).not.toBe("0px");
                 });
 
                 (<any>dataView.metadata.objects).outline.show = false;
                 visualBuilder.updateFlushAllD3Transitions(dataView);
 
                 visualBuilder.dots.forEach((element: HTMLElement) => {
-                    assertColorsMatch(element.style.fill, element.style.stroke);
+                    expect(element.style.strokeWidth).toBe("0px");
                 });
             });
+
+            it("stroke width", () => {
+                const strokeWidth: number = getRandomNumber(1, 5);
+
+                (<any>dataView.metadata.objects).outline.strokeWidth = strokeWidth;
+
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                visualBuilder.dots.forEach((element: HTMLElement) => {
+                    expect(element.style.strokeWidth).toBe(`${strokeWidth}px`);
+                });
+            });
+
+            it("stroke before and after apply of high-contrast", () => {
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                visualBuilder.dots.forEach((element: HTMLElement) => {
+                    assertColorsMatch(element.style.fill, element.style.stroke, true)
```

Comment:
remove third argument _true_ value if it is the default value (check source code)
do this for all similar cases

Reply:
it's default value is false, so it is definitely will stay the same